### DISCUSSION TITLE
Google is detecting Next.js build files as malicious on my production site

### DISCUSSION THREAD

**Discussion by peter-stuart-turner** [vercel/next.js#43399] [Help]
I'm not sure why this is the case - but in trying to set up Google ads, we are running into some issues where URLS such as 

https://[MY_DOMAIN]/_next/static/79qK2Z231TnLZLlXBSnbK/_buildManifest.js 

https://[MY_DOMAIN]/_next/static/79qK2Z231TnLZLlXBSnbK/_ssgManifest.js  

https://[MY_DOMAIN]/_next/static/chunks/2430-0ec59fa8727e4e04.js 

https://[MY_DOMAIN]/_next/static/chunks/439-d921f71d2b760f3b.js  

https://[MY_DOMAIN]/_next/static/chunks/4634-f4960f82a3a4e33e.js  

Are marked as malicious - "Post investigating the account. The latest scan of your site detects Malware/links that are potentially harmful to you and to the site visitors."

...amongst about 10/15 more 

Does anyone have any ideas on how I should go about investigating/fixing this? Any advice would be much appreciated.

**Comment by dhavaln** [comment]
Were you able to solve this issue? We have the same issue.
